### ISSUE TITLE
Add required contrast in color-contrast message

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#381]
The color contrast message is as follows:

```
"Element has insufficient color contrast of {{=it.data.contrastRatio}} (foreground color: {{=it.data.fgColor}}, background color: {{=it.data.bgColor}}, font size: {{=it.data.fontSize}}, font weight: {{=it.data.fontWeight}})",
```

It would be better if this would include what contrast ratio was required (4.5:1 or 3:1).

**Comment by bradarv90** [comment]
Are you all wanting a PR to fix this?

**Comment by WilcoFiers** [comment]
If you've got one, or a little time to create it, you bet!

**Comment by bradarv90** [comment]
Ok. So I've been looking into this. How do I go about testing that the data is properly being displayed in the message?

**Comment by WilcoFiers** [comment]
You can run `grunt dev` which will start a server, and play around with some code in test/playground.html.

**Comment by WilcoFiers** [comment]
Issue resolved in https://github.com/dequelabs/axe-core/pull/562
